miner/worker.go

```diff
@@ -908,6 +909,19 @@ func (w *worker) commitTransactions(txs *types.TransactionsByPriceAndNonce, coin
 	return false
 }
 
+func (w *worker) fetchPendingL1Messages() []types.L1MessageTx {
```

Comment:
```suggestion
func (w *worker) collectPendingL1Messages() []types.L1MessageTx {
```

`fetch` sounds like we're fetching them from some remote API